scss/bootstrap/vendor/select2/select2-bootstrap.scss

```diff
@@ -1,94 +1,257 @@
 .form-control .select2-choice {
-    border: 0;
-    border-radius: 2px;
-    text-decoration: none !important;
+  border: 0;
+  border-radius: 2px;
+  text-decoration: none !important;
 }
 
 .form-control .select2-choice .select2-arrow {
-    border-radius: 0 2px 2px 0;
-    background: transparent;
-    border-left: 0 none;
-    right: 4px;
-    top: 2px;
+  border-radius: 0 2px 2px 0;
+  background: transparent;
+  border-left: 0 none;
+  right: 4px;
+  top: 2px;
 }
 
-.form-control.select2-container {
-    padding: 0;
-    height: auto;
```

Comment:
I believe we can make those selctors:
.form-control.select2-container,
.form-group .select2-container { }
as main and than use nesting